content/docs/demos/ingress_contour_demo.md

```diff
@@ -2,6 +2,7 @@
 title: "Ingress with Contour"
 description: "HTTP and HTTPS ingress with Contour ingress controller"
 type: docs
+weight: 3
```

Comment:
We will need to update this doc as well to ensure the osm-namespace and mesh name is parameterized like the other demo docs, mainly on line 21

Reply:
Yes, but this doc leverages install time flags, so hardcoding the namespace is fine given that the demo requires install to happen.